dlq_processor/cloudwatchlogs_lambda.js

```diff
@@ -33,9 +33,9 @@ function getConfig(env, errorHandler) {
 
         // The following parameters override the sourceCategoryOverride, sourceHostOverride and sourceNameOverride metadata fields within SumoLogic.
```

Comment:
in this comment, it should be "sourceCategory" instead "sourceCategoryOverride",etc.

Reply:
done